ga4gh/frontend.py

```diff
@@ -23,8 +23,46 @@
 app = flask.Flask(__name__)
 
 
-class ServerStatus(object):
+class Version(object):
+    """
+    A major/minor/revision version tag
+    """
+    @classmethod
+    def parseString(cls, versionString):
+        versions = versionString.strip('vV').split('.')
+        return Version(*versions)
+
+    def __init__(self, major, minor, revision):
```

Comment:
I think the class is useful, but you're right that replacing the major/minor/version attributes with a tuple would allow us to eliminate a lot of the methods.